replace_refs.py

```diff
@@ -0,0 +1,30 @@
+#! /usr/bin/python3
+import argparse
+from argparse import RawTextHelpFormatter
+import linecache
+
+description="replaces the REF allele with the old REF allele if REF = ALT in the remapped VCF"
+
+parser = argparse.ArgumentParser(description = description, formatter_class=RawTextHelpFormatter)
+parser.add_argument("-i", "--vcf", help="VCF file containing remapped variants")
+parser.add_argument("-r", "--oldrefalleles", help="file containing the same header as the VCF file + the old REF \
+        alleles in the same order as the VCF")
+parser.add_argument("-o", "--outfile", help="name of new file")
+args = parser.parse_args()
+
+out_vcf=open(args.outfile, 'w')
+
+with open(args.vcf, 'r') as vcf:
+        for(i, line) in enumerate(vcf):
```

Comment:
Our code style (consistent with PEP 8) is to indent using 4 spaces for each level (not 8 spaces as here)